iomgr/lib/dbengine/parser/Siodb.g4

```diff
@@ -344,25 +343,10 @@ update_stmt:
 		| K_OR K_REPLACE
 		| K_OR K_FAIL
 		| K_OR K_IGNORE
-	)? aliased_qualified_table_name K_SET column_name '=' expr (
-		',' column_name '=' expr
+	)? aliased_qualified_table_name K_SET column_name '=' simple_expr (
```

Comment:
In general, I'd like to clarify more what the problem really is. Let's discuss that in the issue.
I think such fix is incorrect. We have boolean field types, so logical expressions with AND, OR, NOT must be somehow allowed. Also you exclude `function_call` which is only in the `expr`. I think some more sophisticated grammar rework is needed here.